messages/en.json

```
@@ -114,6 +114,7 @@
     "fontFamily": "Font Family",
     "lineHeight": "Line Height",
     "normalizeOnAutoComplete": "Normalize on Auto-Complete",
+    "enableWorkflowExecution": "Enable Workflow Execution (Ctrl+Enter)",
```

Show `Cmd+Enter` on macOS.

```suggestion
    "enableWorkflowExecution": "Execute Workflow on Submit ({key})",
```

Add function to key-utils.ts:

```ts
export function getModKeyName(): string {
  return isMacPlatform() ? "Cmd" : "Ctrl";
}
```